packages/augur-ui/src/modules/alerts/components/alerts-view.styles.less

```diff
@@ -91,26 +91,24 @@
 .dismissContainer {
   align-items: flex-end;
 
-  background: @color-contrast-background-theme;
+  background: #201E22;
```

Comment:
stuff for colors effort to clean up, right?
we are in transition not sure how to rem to fix this.